votes/models.py

```diff
@@ -1880,3 +1884,25 @@ def get_same_day_decision_navigation(decision: DecisionType) -> NavigationResult
             previous_decision = same_day_decisions[current_index + 1]
 
     return NavigationResult(next_decision, previous_decision)
+
+
+class BulkAPIUser(DjangoVoteModel):
+    """
+    Model to track users who have access to bulk API data via token authentication.
+    """
+
+    email: EmailField
+    token: str = ""
```

Comment:
I realise the chance of collision here, given this is a niche service, is extremely low but in theory this should be unique, plus adding an index would make the lookup a bit more efficient